frontend/apps/app/app/(app)/app/projects/[id]/page.tsx

```diff
@@ -0,0 +1,21 @@
+import { ProjectDetailPage } from '@/features/projects/pages'
+import { migrationFlag } from '@/libs'
+import { notFound } from 'next/navigation'
+import React from 'react'
+
+type PageProps = {
+  params: Promise<{
+    id: string
+  }>
+}
+
+export default async function Page({ params }: PageProps) {
+  const { id } = await params
+  const migrationEnabled = await migrationFlag()
+
+  if (!migrationEnabled) {
+    notFound()
+  }
```

Comment:
This process is unnecessary because the page is checked by middleware.ts before reaching the page.

https://github.com/liam-hq/liam/blob/0820ad86d21c6e40a8eccca9f72a51261d955cdd/frontend/apps/app/middleware.ts#L13-L19

```suggestion
```